apps/ui-kit/src/lib/components/molecules/card/CardAction.tsx

```diff
@@ -58,12 +62,13 @@ export function CardAction({
         return (
             <div className="shrink-0">
                 <Button
-                    type={ButtonType.Outlined}
+                    type={buttonType}
```

Comment:
I had the same [error](https://github.com/iotaledger/iota/actions/runs/12084123164/job/33698887069#step:11:1057) in my PR.

And resolved it in the same way for now. But I'd set default type instead of have undefined.
```suggestion
                    type={buttonType || ButtonType.Outlined}
```
this way worked for me.

Reply:
Or alternative way we can avoid using CardAction and use Button directly.